src/mirror.c

```diff
@@ -259,6 +249,7 @@ void handle_mirror_if_stale(void)
 enum swupd_code mirror_main(int argc, char **argv)
 {
 	int ret = SWUPD_OK;
```

Comment:
should `int ret` here be changed to `enum swupd_code ret`?

Reply:
This function mixes int with enum swupd_code ret. It's not ideal, but as I didn't fix that, I let the type of ret to int.

Reply:
ok